src/components/DataDimension/DraggableTransferOption.js

```diff
@@ -13,9 +14,10 @@ const DraggableTransferOption = ({
     disabled,
     onDoubleClick,
 }) => {
+    const data = { label, value, type: EXPRESSION_TYPE_DATA }
     const { attributes, listeners, setNodeRef, transform } = useSortable({
         id: value,
-        data: { value, label, type, index: -1 },
```

Comment:
why no more `index: -1`?

Reply:
I couldn't any situation where it is actually used.